app/controllers/projects_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ProjectsController < ApplicationController
   def index
-    @projects = Project.includes(:categories).active
+    @projects = Project.paginate(page: params[:page],
+    							 per_page: 10).
+    							includes(:categories).active
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Tab detected.